qtpy/__init__.py

```diff
@@ -251,3 +264,12 @@ class PythonQtWarning(Warning):
         "set the 'QT_API' env var to 'pyqt', 'pyqt4' or 'pyside'.",
         DeprecationWarning,
     )
+else:
+    if LooseVersion(QT_VERSION) < LooseVersion(QT_VERSION_MIN):
+        _warn_old_minor_version('Qt', QT_VERSION, QT_VERSION_MIN)
+    if PYQT_VERSION and (LooseVersion(PYQT_VERSION)
+                         < LooseVersion(PYQT_VERSION_MIN)):
+        _warn_old_minor_version('PyQt', PYQT_VERSION, PYQT_VERSION_MIN)
```

Comment:
Maybe we should put here `PyQt5`?

Reply:
I actually thought about that, but I made it `PyQt` to avoid repeating the major version twice (`PyQt5 5.9`), to match the constants and so that we wouldn't need to change it if we check for a minimum PyQt6 version. But I can change it if you're sure.

Reply:
That makes sense, lets leave it as it is then. Thanks for claryfing @CAM-Gerlach :+1: